components/Session/Voting/Vote/Content.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import parse from 'html-react-parser';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
+import ButterToast, { Cinnamon, POS_TOP, POS_RIGHT } from 'butter-toast';
+
+import NavLinks from '../Shared/NavLinks';
+import Thumbs from '../Shared/Thumbs';
+import Stats from '../Shared/Stats';
+
+const MarkdownIt = require('markdown-it');
+
+const MainContent = styled.div`
+  margin-top: 0rem;
+`;
+
+const Section = styled.div`
+  margin-bottom: 3rem;
+
+  ul {
+    margin-top: 0;
+  }
+`;
+
+const DetailsHeader = styled.div`
+  font-size: 2rem;
+`;
+
+const GET_SESSIONS = gql`
+  query getVotingSessions($eventId: ID!) {
+    sessions {
+      me {
+        voting(eventId: $eventId) {
+          isVotingOpen
+          totalSubmitted
+          unVoted {
+            id
+            title
+            longDescription
+            takeaways
+          }
+        }
+      }
+    }
+  }
+`;
+
+const CAST_VOTE = gql`
+  mutation castVote($eventId: ID!, $vote: VoteInput!) {
+    sessions {
+      voting(eventId: $eventId) {
+        cast(vote: $vote) {
+          id
+          notes
+          memberId
+        }
+      }
+    }
+  }
+`;
+
+const Content = () => {
+  const {
+    loading: sessionsLoading,
+    error: sessionsError,
+    data: sessionsData,
+  } = useQuery(GET_SESSIONS, {
+    variables: { eventId: process.env.CURRENT_EVENT_ID },
+  });
+
+  if (sessionsLoading) return null;
```

Comment:
Be good to use the `LoadingIndicator` here too so that there can be something on the screen for the user while this is loading.